### PR TITLE
Update ReadMe

### PR DESCRIPTION
**Pull request by cra0zy**
Just some minor modifications I thought should be done:
 - mention what graphics backends platforms use, for now it's all OpenGL, but I would really like to see us use Vulkan and Metal :)
 - combine submodule commands into 1
 - correct command for generating platform solutions from Windows

CC. @tomspilman

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ If you are interested in contributing fixes or features to MonoGame, please read
 The full source code is available here from GitHub:
 
  * Clone the source: `git clone https://github.com/mono/MonoGame.git`
- * Setup the submodules: `git submodule init` and `git submodule update`
+ * Setup submodules (this can be skipped by running Protobuild.exe): 'git submodule update --init'
```
Comment (tomspilman) [inline README.md]: I really don't want to promote this aspect of Protobuild.

Comment (cra0zy) [inline README.md]: Updated

File: README.md [hunk 2]
```diff
@@ -82,7 +82,9 @@ A high level breakdown of the components of the framework:
  * The [Tools/Pipeline](Tools/Pipeline) tool is a GUI frontend for content processing.
 
 
-You can generate solutions for platforms that are not buildable from the current OS with: `mono Protobuild.exe --generate $PLATFORM`.
+You can generate solutions for platforms that are not buildable from the current OS with: 
+ - Windows: `.\Protobuild.exe --generate $PLATFORM`
+ - Linux or Mac: `mono Protobuild.exe --generate $PLATFORM`
```
Comment (tomspilman) [inline README.md]: Move this up into the first list in this section right after the first Protobuild mention.

Comment (cra0zy) [inline README.md]: Done :)

**Comment by tomspilman** [comment]
Thanks @cra0zy!

**Pull request merged**